src/main/java/org/dataone/bookkeeper/jdbi/QuotaStore.java

```diff
@@ -182,8 +182,7 @@
        "unit = :unit, " +
        "subscriptionId = :subscriptionId, " +
        "subject = :subject " +
-       "WHERE id = :id " +
-       "RETURNING id")
+       "WHERE id = :id ")
```

Comment:
I assume we are able to drop the `RETURNING id` clause because we are binding the row to a `Quota` object, and so don't need to return the `id` for a later call to `SELECT`.  I wonder if we need the `@getGeneratedKeys` annotation either.  My hunch is we don't, and if so (i.e. the `id` is returned in the quota row), let's remove this annotation for clarity.